server.js

```diff
@@ -92,6 +92,12 @@ app.use(function (req, res, next) {
   next();
 });
 
+// Set Cache-Control to no-store to stop states from being cached.
+app.use(function (req, res, next){
+  res.setHeader("Cache-Control", "no-cache,no-store,max-age=0");
```

Comment:
Yeah it's a bit neater. IMO setting headers is as distinct as the functionality needs to be. Just update the comment on line 88 to `Setting headers` or something then it makes sense to group them.